scripts/build-data.js

```diff
@@ -69,6 +69,7 @@ const envMap = {
   iojs: "node3.3",
   node64: "node6",
   node65: "node6.5",
+  node76: "node7.6",
```

Comment:
Yeah, still in the thick of a refactor that hopefully will remove the need for this. (fingers crossed)